src/webextension/unlock/components/app.css

```diff
@@ -5,70 +5,40 @@
 body {
   padding: 0;
   margin: 0;
-  width: 344px;
+  width: 320px;
   color: #0c0c0d;
   background-color: #f9f9fa;
```

Comment:
@jimporter I think this needs to change to`#ededf0` from `#f9f9fa`?

![firefox_nightly_and_d_1_1_0_default_-_locked_state](https://user-images.githubusercontent.com/49511/36911082-bb9af14c-1dff-11e8-94eb-1f76bc1e72af.png)

Reply:
Fixed. (I wasn't testing with the real doorhanger; I usually just open up the page directly in the browser, since it's faster to get to.)